test/AWS.Deploy.CLI.IntegrationTests/BlazorWasmTests.cs

```diff
@@ -53,8 +53,7 @@ public BlazorWasmTests()
         }
 
         [Theory]
-        [InlineData("testapps", "BlazorWasm31", "BlazorWasm31.csproj")]
-        [InlineData("testapps", "BlazorWasm50", "BlazorWasm50.csproj")]
+        [InlineData("testapps", "BlazorWasm60", "BlazorWasm60.csproj")]
```

Comment:
Should we leave 3.1 and 5 under test since we're not removing support for deploying them as far as I understand?

Reply:
I think the value of testing each framework as an integ test is minimal. We don't do anything different for whatever framework being targeted. I'm fine not having the 3.1 and 5.0 tests.